src/sections/description.liquid

```diff
@@ -1,10 +1,20 @@
-<section class="description {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="description">
+<section class="description large-section {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="description">
```

Comment:
not stoked on the added classname here

Reply:
would recommend refactoring each of those section-large etc declarations in base-utils to more like
section__large (etc)

Reply:
should be pretty quick work with find all

Reply:
Addressed in: https://github.com/thinkific/fundamentals/pull/4